tests/lib/external/snapd-testing-tools/tests/os.query/task.yaml

```diff
@@ -70,19 +70,19 @@ execute: |
             ! os.query is-ubuntu 20.04
             ! os.query is-core
             ;;
-        ubuntu-23.04-64)
+        ubuntu-23.10-64)
             os.query is-classic
-            os.query is-ubuntu 23.04
+            os.query is-ubuntu 23.10
             ! os.query is-ubuntu 21.04
             ! os.query is-core
             ;;
-        ubuntu-23.10-64)
+        ubuntu-24.04-64)
             os.query is-classic
-            os.query is-ubuntu 23.10
+            os.query is-ubuntu 24.04
             ! os.query is-ubuntu 21.04
             ! os.query is-core
             ;;
-        ubuntu-24.04-64)
+        debian-11-64)
```

Comment:
I think this option still shouldn't be here, since it's checking that `debian-11-64` `is-ubuntu 24.04`. AFAICT the `debian-11-64` below is the correct one and this whole case can be deleted.

Here's the problem:

https://github.com/snapcore/snapd/blob/4166485f5c379366e0913ee3624e4063c734ccef/tests/lib/external/snapd-testing-tools/tests/os.query/task.yaml#L79-L96